src/osd/ReplicatedBackend.cc

```diff
@@ -833,6 +841,9 @@ int ReplicatedBackend::be_deep_scrub(
     }
   }
 
+  // we have the full omap now. Finalize the perf counting
+  perf_logger.inc(io_counters.omapget_bytes, pos.omap_bytes);
```

Comment:
Why not just increment perf_logger originally and skip io_counters entirely?

Reply:
see below